gapic/schema/wrappers.py

```diff
@@ -821,13 +822,64 @@ def field_headers(self) -> Sequence[str]:
 
         return next((tuple(pattern.findall(verb)) for verb in potential_verbs if verb), ())
 
+    def http_rule_to_tuple(self, http_rule: http_pb2.HttpRule) -> Tuple[str, str, str]:
+        """Represent salient info in an http rule as a tuple.
+
+        Args:
+          http_rule: the http option message to examine.
+
+        Returns:
+          A tuple of (method, uri pattern, body or None),
+            or None if no method is specified.
+        """
+
+        http_dict: Mapping[str, str]
+
+        method = http_rule.WhichOneof('pattern')
+        if method is None or method == 'custom':
+            return ('', '', '')
+
+        uri = getattr(http_rule, method)
+        if not uri:
+            return ('', '', '')
+        body = http_rule.body if http_rule.body else None
```

Comment:
Outrageous nit, feel free to ignore: can simplify this into
```python
body = http_rule.body or None
```

Reply:
Done.